x-pack/plugins/synthetics/server/routes/monitor_cruds/add_monitor.ts

```diff
@@ -152,13 +154,23 @@ export const hydrateMonitorFields = ({
     string,
     { preserve_namespace?: boolean }
   >;
+
   return {
     ...normalizedMonitor,
     [ConfigKey.MONITOR_QUERY_ID]: normalizedMonitor[ConfigKey.CUSTOM_HEARTBEAT_ID] || newMonitorId,
     [ConfigKey.CONFIG_ID]: newMonitorId,
     [ConfigKey.NAMESPACE]: preserveNamespace
       ? normalizedMonitor[ConfigKey.NAMESPACE]
       : getMonitorNamespace(server, request, normalizedMonitor[ConfigKey.NAMESPACE]),
+    [ConfigKey.SOURCE_PROJECT_CONTENT]: !!(normalizedMonitor as BrowserSensitiveSimpleFields)?.[
+      ConfigKey.SOURCE_INLINE
+    ]
+      ? await inlineToProjectZip(
+          (normalizedMonitor as BrowserSensitiveSimpleFields)?.[ConfigKey.SOURCE_INLINE] as string,
+          newMonitorId,
+          server.logger
+        )
+      : '',
```

Comment:
This looks a bit polluted. What do you think of extracting `(normalizedMonitor as BrowserSensitiveSimpleFields)?.[ConfigKey.SOURCE_INLINE] as string | undefined` to a const and reuse it on line 165 and 169?

```suggestion
    [ConfigKey.SOURCE_PROJECT_CONTENT]: !!sourceLine
      ? await inlineToProjectZip(
          sourceLine,
          newMonitorId,
          server.logger
        )
      : '',
```

Reply:
++, implemented [45246c2](https://github.com/elastic/kibana/pull/176094/commits/45246c26b222093c4314506b0e60c406d357f2fd)